shell/platform/embedder/tests/embedder_vk_unittests.cc

```diff
@@ -96,12 +96,12 @@ static_assert(
 }  // namespace
 
 TEST_F(EmbedderTest, CanSwapOutVulkanCalls) {
-  auto& context = GetEmbedderContext(EmbedderTestContextType::kVulkanContext);
   fml::AutoResetWaitableEvent latch;
+
+  auto& context = static_cast<EmbedderTestContextVulkan&>(
+      GetEmbedderContext(EmbedderTestContextType::kVulkanContext));
   context.AddIsolateCreateCallback([&latch]() { latch.Signal(); });
-  EmbedderConfigBuilder builder(context);
-  builder.SetVulkanRendererConfig(
-      SkISize::Make(1024, 1024),
+  context.SetVulkanInstanceProcAddressCallback(
```

Comment:
Vulkan-specific context configuration now occurs on the Vulkan-specific context.